app.js

```diff
@@ -10,6 +10,11 @@
 var express = require('express');
 var db = require('./db');
 var app = express();
+var adapterManager = require('./adapter-manager.js');
+var bodyParser = require('body-parser');
```

Comment:
It parses the body as JSON, so that you can use request.body['led'] or whatever the attribute name is. We could parse the body ourselves.